features/admin.extensions.v1/configs/common.tsx

```diff
@@ -61,6 +63,35 @@ export const commonConfig: CommonConfig = {
                         path: AppConstants.getPaths().get("USERSTORE_TEMPLATES"),
                         protected: true,
                         showOnSidePanel: false
+                    },
+                    {
+                        component: lazy(() =>
+                            import("@wso2is/admin.remote-userstores.v1/pages/remote-customer-user-store-create-page")
+                        ),
+                        icon: {
+                            icon: getSidePanelIcons().childIcon
+                        },
+                        id: "remote-user-store-create",
+                        name: "userstore create",
+                        path: RemoteUserStoreConstants.getPaths().get("REMOTE_USER_STORE_CREATE"),
+                        protected: true,
+                        showOnSidePanel: false
+                    },
+                    {
+                        component: lazy(() =>
+                            import("@wso2is/admin.remote-userstores.v1/pages/remote-user-store-edit-page")
+                        ),
+                        exact: true,
+                        icon: {
+                            icon: getSidePanelIcons().childIcon
+                        },
+                        id: "remote-edit-user-store",
+                        name: "console:manage.features.sidePanel.editUserstore",
```

Comment:
Doesn't the `userstores` namespace have a side panel translation?
Also since this is not displayed on the side panel, you can skip the name.

Reply:
Addressed with 0bc853333b14313614be9c0e79c2dd4a88d30ea0.